doc/performance_counters.rst

```diff
@@ -54,16 +56,22 @@ In particular, to enable/disable ``mcycle(h)``, bit 0 must be written. For ``min
 The lower 32 bits of all counters can be accessed through the base register, whereas the upper 32 bits are accessed through the ``h``-register.
 Reads to all these registers are non-destructive.
 
-Counter Config
---------------
+Parametrization at synthesis time
+---------------------------------
+
+The ``mcycle(h)`` and ``minstret(h)`` counters are unconditionally enabled and 64 bit wide.
 
-The ``mcycle(h)`` and ``minstret(h)`` counters are 64 bit wide.
-The bit width of the event counters ``mhpmcounter3(h)`` - ``mhpmcounter31(h)`` can be controlled via the ``WidthMHPMCounters`` parameter.
+The event counters ``mhpmcounter3(h)`` - ``mhpmcounter31(h)`` are parametrizable.
+Their width can be parametrized between 32 and 64 bit through the ``WidthMHPMCounters`` parameter, which defaults to 40 bit wide counters.
```

Comment:
Actually, also widths smaller than 32 bit are supported in the RTL.

Reply:
Ah ok, then `integration.rst` had it wrong (fixed that), and we probably had this limitation in a previous RTL version. (I should double-check more and rely less on my memory ...)